source/getting-started/installation-raspberry-pi.markdown

```diff
@@ -1,29 +1,207 @@
 ---
 layout: page
-title: "Installation on a Raspberry Pi"
-description: "Instructions to install Home Assistant on a Raspberry Pi."
-date: 2016-04-16 11:36
+title: "Raspbian Lite on a Raspberry Pi"
+description: "Instructions to install Home Assistant on a Raspberry Pi runnning Raspbian Lite."
+date: 2016-09-05 16:00
 sidebar: true
 comments: false
 sharing: true
 footer: true
 ---
 
-Home Assistant requires the Raspberry Pi to run [Raspbian Jessie](https://www.raspberrypi.org/downloads/raspbian/). This version was released on September 24, 2015 and comes by default with Python 3.4 which is required for Home Assistant.
+### {% linkable_title Installation %}
 
-Execute the following code in a console:
+This installation of Home Assistant requires the Raspberry Pi to run [Raspbian Jessie Lite](https://www.raspberrypi.org/downloads/raspbian/).
+The installation will be installed in a [Virtual Environment](/getting-started/installation-virtualenv) with minimal overhead. Instructions assume this is a new installation.
 
+Connect to the Raspberry Pi over ssh. Default password is `raspberry`.
+```bash
+$ ssh pi@ipadress
+```
+
+Changing the default password is encouraged.
+```bash
+$ passwd
+```
+
+Update the system.
+```bash
+$ sudo apt-get update
+$ sudo apt-get upgrade -y
+```
+
+Install the dependencies.
+```bash
+$ sudo apt-get install python3 python3-venv python3-pip python3-sphinx
+$ sudo pip3 install virtualenv
```

Comment:
If one is going to switch python environments frequently I'd recommend also installing `virtualenvwrapper`. It'll simplify switching to a virtual env by use of the `workon` command. This might be more a target for developers though, but I thought I'd mention it.